### ISSUE TITLE
results on referformer

### ISSUE THREAD

**Issue by dengandong** [henghuiding/MeViS#7]
Thanks for the excellent work. In table 5 in the paper, the ReferFormer reaches 31.0 J&F on your dataset and how are the results obtained? Is it directly evaluated on your validation set without training (i.e. directly using pretrained referformer) or evaluated after training with the training set?

**Comment by henghuiding** [comment]
Hi,

Thank you very much for your interest.

It is evaluated after training with the MeViS training set.
